finagle-mysql/src/main/scala/com/twitter/finagle/mysql/transport/MysqlBuf.scala

```diff
@@ -132,6 +132,30 @@ class MysqlBufReader(buf: Buf) extends ProxyByteReader {
       case len => throw new IllegalStateException(s"Invalid length byte: $len")
     }
   }
+
+  private val unsignedLongMaxValue: BigInt = BigInt("18446744073709551615")
+
+  def readUnsignedLongLE(): BigInt = {
```

Comment:
Both of these seem generally useful. Maybe we should move them under `ByteReader`, where we have O(1) get?